programs/dialect/src/lib.rs

```diff
@@ -74,6 +74,9 @@ pub mod dialect {
         encrypted: bool,
         scopes: [[bool; 2]; 2],
     ) -> ProgramResult {
+        if !(scopes[0][0] || scopes[1][0]) {
```

Comment:
What do yo think about enforcing that `owner` is the admin?

Reply:
Agreed, updated